.github/workflows/check-network.yaml

```diff
@@ -46,7 +42,10 @@ jobs:
         with:
           ref: ${{ env.github_ref }}
 
-      - name: Set up Go 1.x
+      - name: Prepare build environment
+        run: .github/workflows/prepare-build-env.sh
```

Comment:
hmm, not sure if I'm getting blind but I do not see this file being part of the PR?

Reply:
oh, never mind, it came already with some earlier PR and now you're wiring it up to be used everywhere 👍

Reply:
This is part of the "Remove as many hard-coded go versions as possible" effort. It has already been done for the main build workflow. I thought it would be better to propagate that technique to the remaining workflows, instead of updating the hard-coded go versions there.